server-toss/tests/load-test/README.md

```
@@ -0,0 +1,42 @@
+  - `SEED_DRAWING_USER_COUNT=1000`
+  - `SEED_DRAWING_BATCH_SIZE=200`
+
+```sh
+# 기본 1,000명
+pnpm -F server-toss seed:drawing:large
+```
+
+## 부하 테스트 가이드
+
+> 도커 컨테이너로 하는 이유는 컨테이너 실행 시 cpu, memory 자원 제한을 두기 위함입니다.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)